.github/workflows/ci.yml

```diff
@@ -0,0 +1,15 @@
+name: CI
+on: push
+jobs:
+  style:
```

Comment:
Could you update these to use the centralised workflows? An example can be found in PlaceOS/rest-api